### ISSUE TITLE
Set keyboard focus to the first paragraph or interactable control for popup dialogs

### ISSUE THREAD

**Issue by Flameborn** [bookwyrm-social/bookwyrm#1041]
Currently, when a dialog pops up (e.g. when adding a new reading progress, or when finishing a book on the feeds page), keyboard focus usually stays on the main page and there is no indication that something appeared on-screen.

This can be misleading to screen reader users, since they can navigate away, believing that no additional input is required.

A way to solve this is to move the keyboard focus to the first control that can be interacted with (except for the close/OK/Save buttons, in which case focus the second control). If only one control is present, move focus to it, or the first paragraph of text.

**Comment by mouse-reeve** [comment]
This should be happening in most places, and happily the markup is designed to support this. I see where it's not working with reading progress on the book page, but I'm not seeing the problem with finishing a book from the feed page. Can you give me more detail about that one?

**Comment by mouse-reeve** [comment]
I think this has been resolved.
